library/Odesk/Phystrix/RequestLog.php

```diff
@@ -49,4 +49,72 @@ public function addExecutedCommand(AbstractCommand $command)
     {
         $this->executedCommands[] = $command;
     }
+
+    /**
+     * Formats the log of executed commands into a string usable for logging purposes.
```

Comment:
would you mind adding an example of what the output it produces to the doc block ?